db/migrate/20160415121724_create_cloud_services.rb

```diff
@@ -0,0 +1,21 @@
+class CreateCloudServices < ActiveRecord::Migration[5.0]
+  def change
+    create_table :cloud_services do |t|
+      t.string :ems_ref
+      t.string :source
+      t.string :binary
+      t.string :host_name
+      t.string :status
+      t.string :scheduling_status
+      t.string :zone_name
+      t.string :disabled_reason
```

Comment:
There is a disabled_reason, so I would expect to see a boolean column called `disabled`.  Is that not needed?